gocd-agent/agent.nuspec

```diff
@@ -20,6 +20,11 @@ The following install arguments can be set:
 These parameters can be passed to the installer with the use of `--ia`.
     </description>
     <projectUrl>https://gocd.io/</projectUrl>
+    <docsUrl>https://docs.gocd.io</docsUrl>
+    <mailingListUrl>https://www.gocd.io/subscribe/</mailingListUrl>
+    <bugTrackerUrl>https://github.com/gocd/gocd/issues</bugTrackerUrl>

```

Comment:
Can you please change this to be `https://github.com/gocd/gocd-chocolatey/issues`